exp-player/addon/components/exp-card-sort/template.hbs

```diff
@@ -78,13 +78,14 @@
                         {{#each group.categories as |category|}}
                             <div class="col-sm-4 small-bucket
                                 {{unless (eq category.cards.length category.max) 'text-danger validation-error'}}
-                                ">
-                                <h5 class="bucket-label visible-xs {{if
+                                {{if (eq category.cards.length category.max) 'text-success'}}
+                                " >
```

Comment:
Super-mega-nitpick: the added space between the `"` and the `>` makes the diff bigger.